Km_to_Miles.java

```diff
@@ -0,0 +1,20 @@
+class Km_to_Miles extends ConsoleProgram {
+
+  /**
+  * Description
+  * @author:
+  */
+  
+  public void run() {
+    
+    double doubleKilometers;
+    double doubleMiles;
+
+    doubleKilometers = readDouble("Enter number of kilometers: ");
+
+    doubleMiles = doubleKilometers * 0.621371;
```

Comment:
should've made the conversion number shorter so it doesn't output an absurdly long number